webtorrent/tracker_client.go

```diff
@@ -130,9 +133,7 @@ func (tc *TrackerClient) Run() error {
 func (tc *TrackerClient) Close() error {
 	tc.mu.Lock()
 	tc.closed = true
-	if tc.wsConn != nil {
-		tc.wsConn.Close()
```

Comment:
Probably still need this.

Reply:
https://github.com/anacrolix/torrent/pull/505